src/server/console/default_configuration.cpp

```diff
@@ -105,8 +106,11 @@ std::shared_ptr<mir::ConsoleServices> mir::DefaultServerConfiguration::the_conso
                 {
                     try
                     {
-                        auto const vt_services = std::make_shared<mir::LogindConsoleServices>(
-                            std::dynamic_pointer_cast<mir::GLibMainLoop>(the_main_loop()));
+                        auto session_lock_instance = the_session_lock();
+                        auto const vt_services = mir::LogindConsoleServices::create(
+                            std::dynamic_pointer_cast<mir::GLibMainLoop>(the_main_loop()),
+                            session_lock_instance);
+                        session_lock_instance->register_interest(vt_services);
```

Comment:
The point of introducing `LogindConsoleServices::create()` is that it does this initialization
```suggestion
```